source/hic_hal/stm32/stm32h743ii/extended_features/include/udb_extended_features_task.h

```diff
@@ -0,0 +1,20 @@
+/*
+ *  Description:
+ *    Create an auxiliary task so that we don't need to put UDB's logic in the main task.
+ *
+ */
+#ifndef _UDB_EXTENDED_FEATURES_TASK_INCLUDED_H_
+#define _UDB_EXTENDED_FEATURES_TASK_INCLUDED_H_
+
+#include "cmsis_os2.h"
+
+#define UDB_EXTENDED_FEATURES_TASK_STACK          (800)
+#define UDB_EXTENDED_FEATURES_TASK_PRIORITY       (osPriorityNormal)
+#define UDB_EXTENDED_FEATURES_TASK_FLAGS_3S       (1 << 0)
+
+// 1 tick is 10ms
```

Comment:
Is the 10ms tick a fixed value? If not, is there a way to retrieve the tick rate and define this value in terms of the tick rate?